### DISCUSSION TITLE
text position in tasks/parallel tasks

### DISCUSSION THREAD

**Discussion by pitecantropo** [csgoh/roadmapper#65] [Ideas]
Hey, 
This is a very neat project!

Would be nice to be able to override the "center alignment" text in the tasks and parallel tasks. 
The reason being that sometimes the milestone covers the text if it happens to be coincide with the middle of the bar.
Another alternative to circumvent could be the placement of the milestone marker above the bar instead of over it?

Many thanks in any case for this tool and any consideration!

**Comment by csgoh** [comment]
Hi @pitecantropo, the text aligment for tasks and parallel tasks already supported in the current version.
Just add `text_alignment` as the parameter.

<img width="941" alt="image" src="https://github.com/csgoh/roadmapper/assets/20867152/4910956e-914a-4841-bb62-bd30450c3bf5">

Supported `text_alignment` are:
* left
* centre
* right

Example:
```python
group = roadmap.add_group("Core Product Work Stream", text_alignment="left")
task = group.add_task("Base Functionality", "2022-11-01", "2023-10-31", text_alignment="left")
task.add_milestone("v.1.0", "2023-02-15")
task.add_milestone("v.1.1", "2023-08-01")
parellel_task = task.add_parallel_task("Enhancements", "2023-11-15", "2024-03-31", text_alignment="left")
```

**Reply by pitecantropo** [reply]
That perfect. thanks!
Had missed it.
